src/Aquifer.API/Endpoints/Resources/Content/MachineTranslation/Create/Endpoint.cs

```diff
@@ -17,6 +18,8 @@ public override void Configure()
 
     public override async Task HandleAsync(Request req, CancellationToken ct)
```

Comment:
As I understand it this endpoint is going away entirely so probably not worth making changes to it

Reply:
Is Retranslate going away too?